### PR TITLE
Add a section about Polyup to the migration guide

### PR DESCRIPTION
**Pull request by rictic**
I wasn't sure where to put it. I went with prominent placement at the top of the page to catch people who visit the page frequently and would likely otherwise miss it.

### CODE REVIEW

File: 1.0/docs/migration.md [hunk 1]
```diff
@@ -11,6 +11,16 @@ subtitle: About this release
 This guide describes the changes required to migrate a {{site.project_title}} 
 element from 0.5 to 1.0.  
 
+## Polyup
```
Comment (ebidel) [inline 1.0/docs/migration.md]: How about "Polyup tool"

File: 1.0/docs/migration.md [hunk 2]
```diff
@@ -11,6 +11,16 @@ subtitle: About this release
 This guide describes the changes required to migrate a {{site.project_title}} 
 element from 0.5 to 1.0.  
 
+## Polyup 
+
+We have been working on a tool called `polyup` to automatically
+perform many of the changes detailed here. See an interactive demo
```
Comment (ebidel) [inline 1.0/docs/migration.md]: detailed -> detailed in this migration guide.

**Comment by ebidel** [comment]
Reviewed

File: 1.0/docs/migration.md (hunk 1 above)
Comment (rictic) [inline 1.0/docs/migration.md]: +1, done

File: 1.0/docs/migration.md (hunk 2 above)
Comment (rictic) [inline 1.0/docs/migration.md]: +1, done.

**Comment by ebidel** [comment]
LGTM. Thanks for this.

**Pull request merged**